src/middlewared/middlewared/plugins/interface/netif_linux/bridge.py

```diff
@@ -1,23 +1,21 @@
-# -*- coding=utf-8 -*-
-import json
-import logging
-
-import middlewared.plugins.interface.netif_linux.interface as interface
+from json import loads
+from logging import getLogger
+from pyroute2 import NDB
 
 from .utils import run
 
-logger = logging.getLogger(__name__)
+logger = getLogger(__name__)
 
 __all__ = ["create_bridge", "BridgeMixin"]
 
 
 def create_bridge(name):
-    cmd = [
-        "ip", "link", "add", name, "type", "bridge",
-        "stp_state", "1"  # enable stp by default 1 == on 0 == off
-    ]
-    run(cmd)
-    interface.Interface(name).up()
+    ndb = None
+    try:
+        ndb = NDB(log='off')
```

Comment:
You can just move this line away from try/catch block and remove `if` condition in `finally` block. Also let's use consistent single/double quotes in a file